README.md

```diff
@@ -1,2 +1,11 @@
 # comp110-worksheet-C
 Base repository for COMP110 worksheet C
+* (a) The algorithm is testing whether there are 2 or more of the same elements in a list.
+* (b) The algorithm worst case is quadratic because it checks each element of i against each element of j n times (from the same list of length n). For every i (n number of times) it goes looping through j n times. O(n * n). The j loop is nested inside the i loop.
+* (c) The algorithm still works because j is still comparing to the elements the i loop has gone through already.
```

Comment:
✔️ but worth spelling out that the original algorithm compares each pair of elements twice (A==B and B==A) whereas this version eliminates the redundant comparisons.